src/amo/components/LandingPage.js

```diff
@@ -108,9 +109,29 @@ export class LandingPageBase extends React.Component {
     }
 
     const { addonType, html } = content;
+    const themeText = 'Change your browser’s appearance. Choose from thousands of themes to give Firefox the look you want.';
+    const extensionsText = 'Install powerful tools that make browsing faster and safer, add-ons make your browser yours.';
 
     return (
       <div className={classNames('LandingPage', `LandingPage-${addonType}`)}>
+
+        <div className="Addon-Header">
+          <Icon name={classNames(`${addonType}`)} className="Addon-Icon" />
+          <div className="Addon-Header-Text">
+            <span className="Addon-Header-Text-Bold">
```

Comment:
Try not to describe the look of the element but instead what element it is and add the look in the CSS. So this should be something like `LandingPage-heading` and then the fact that it is bold is stored in the CSS.

This could probably be an `h1` too.